src/webextension/widgets/button.css

```diff
@@ -71,10 +80,14 @@
 
 .normal-size {
   height: 32px;
-  min-width: 80px;
   padding: 0 8px;
   font-size: 13px;
   border-radius: 2px;
+  min-width: 80px;
+}
+
+.primary-theme.normal-size {
+  min-width: 132px;
```

Comment:
What's this for? Is there a particular button that needs this? As far as I can tell, it's an optional part of the spec for normal-sized buttons (and doesn't necessarily have anything to do with primary-themed buttons).

I'm not sure *all* normal-sized primary buttons and *no* normal-sized non-primary buttons should have this. I could see adding a sub-theme like `normal-wide` (or maybe even just call it `wide`), but as it is, this seems like the wrong way to do it.

Reply:
@jimporter I have all of the other changes pretty much wrapped up except for this one. I'm unsure how to setup a sub-theme with your button themes in the button.js file. Is this something you can run through real quick with me either via Slack or Vidyo? Thanks!

Reply:
If you go to `widgets/button.js`, you'll see this block:

https://github.com/mozilla-lockbox/lockbox-extension/blob/956d5bf56299d677b79c21ac2ff21649b090e961/src/webextension/widgets/button.js#L16-L20

You can add an entry like this to it (and then add a rule for `.wide` with your `min-width` in the CSS file):

```
wide: `${styles.normalSize} ${styles.wideSize}`,
```

Then you just have to find the buttons you want to be wide in the appropriate JS files and add `theme="wide"` to them.

Reply:
Gotcha, thanks for the help!

Reply:
This all seemed good, except you meant to use `size="wide"` instead of `theme="wide"`, correct? As long as that is right I'll get this committed and ready for review.

Reply:
Right, sorry; `size="wide"` is what you want.